Modules/Core/Common/wrapping/itkSpatialFunction.wrap

```diff
@@ -1,7 +1,5 @@
 itk_wrap_class("itk::SpatialFunction")
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
-    foreach(t ${WRAP_ITK_REAL})
-      itk_wrap_template("${ITKM_${t}}${d}${ITKM_PD${d}}" "${ITKT_${t}},${d},${ITKT_PD${d}}")
-    endforeach()
```

Comment:
We will want to keep this if other classes are using them, and just add the new instantiation.

Reply:
Ooops. I simplified a little too much the wrapping files when I only wrapped for `double` instead of all real types. I forgot that this file already existed before this PR...